packages/components/badge/src/badge.vue

```diff
@@ -33,6 +33,8 @@ const ns = useNamespace('badge')
 const content = computed<string>(() => {
   if (props.isDot) return ''
 
+  if (props.value === 0 && !props.showZero) return ''
```

Comment:
The judgment here should be placed on line 39.
```ts
if (props.max < props.value) {
  return `${props.max}+`
}
return props.value === 0 && !props.showZero ? '' : `${props.value}`
```
For example, when `props.value` is 0, `showZero` is true, but `props.max` is `-1`, the badge should be displayed.

Reply:
Like this? 
```suggestion
 const content = computed<string>(() => {
  if (props.isDot) return ''
  if (isNumber(props.value) && isNumber(props.max)) {
    if (props.max < props.value) {
      return `${props.max}+`
    }
    return props.value === 0 && !props.showZero ? '' : `${props.value}`
  }
  return `${props.value}`
})
```

Reply:
Ok